README.md

```diff
@@ -368,6 +368,7 @@ See also [A comparison of operating systems written in Rust](https://github.com/
 * [arthrp/consoletimer](https://github.com/arthrp/consoleTimer) — Simple timer for your terminal. [<img src="https://api.travis-ci.org/arthrp/consoleTimer.svg?branch=master">](https://travis-ci.org/arthrp/consoleTimer)
 * [tversteeg/emplace](https://github.com/tversteeg/emplace) — Synchronize installed packages on multiple machines
 * [myfreeweb/freepass](https://github.com/myfreeweb/freepass) — The free password manager for power users.
+* [yoannfleurydev/gitweb](https://github.com/yoannfleurydev/gitweb) — Open the current remote repository in your browser. [![GitHub Actions](https://github.com/yoannfleurydev/gitweb/workflows/Publish/badge.svg)](https://github.com/yoannfleurydev/gitweb/actions)
```

Comment:
Please add the branch to the github actions image as per https://docs.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#example-using-the-branch-parameter

Reply:
Done. Sorry for the inconvenience.